react/features/base/devices/functions.js

```diff
@@ -0,0 +1,33 @@
+// @flow
+
+import JitsiMeetJS from '../lib-jitsi-meet';
+import { updateSettings } from '../settings';
+
+/**
+ * Get device id of the audio output device which is currently in use.
+ * Empty string stands for default device.
+ *
+ * @returns {string}
+ */
+export function getAudioOutputDeviceId() {
+    return JitsiMeetJS
+        ? JitsiMeetJS.mediaDevices.getAudioOutputDevice() : undefined;
```

Comment:
The truthy check for JitsiMeetJS is concerning to me because I'm assuming before JitsiMeetJS was just defined somewhere, globally, and available but now it can be unavailable and the return value might be different now. From where was this being called that JitsiMeetJS was falsy?

Reply:
Right, that was an earlier implementation when I relied on the global variable, but now I import it so it makes no sense anymore